### DISCUSSION TITLE
PrimeFaces Calendar Inline Mode

### DISCUSSION THREAD

**Discussion by marklambirth** [primefaces/community#38] [PrimeFaces] [answered]
I'm trying to resize the PrimeFaces calendar while in inline mode and I can't figure out how to do it. Google has very little info on this. I've tried referencing .ui-datepicker in CSS and setting the width but that doesn't affect it. Does anyone have any other suggestion? Thanks

**Comment by jepsar** [comment, marked as answer]
Have a look at https://stackoverflow.com/questions/8768317/how-do-i-override-default-primefaces-css-with-custom-styles
